052-SAPAppModernization/Student/Challenge-06.md

```diff
@@ -0,0 +1,32 @@
+# Challenge 06 - Self-service chatbot using data from SAP S/4 HANA system
+
+[< Previous Challenge](./Challenge-05.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-07.md)
+
+## Introduction
+Your existing app service front end poses a nice interface into the SAP world. Often they get deployed or embedded into internal enterprise portals for discovery and governance purposes. In the Microsoft ecosystem there is no place better than the Microsoft Teams Client to achieve this. Speaking of Teams: You want to use natural language to query your SAP? Any interest? In this challenge you will implement a bot service that can leverage your existing infrastructure and process OData calls.
```

Comment:
Consider removing "Speaking of Teams" and appending "from Teams" after "query your SAP?".